UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Leisure.cshtml

```diff
@@ -43,10 +43,10 @@
                     @for (int i = 0; i < showFilesResponseModel.Count; i++)
                     {
                         <tr>
-                            <td id="filename_@i"><a target="_blank" aria-label="@showFilesResponseModel[i].FileDescription. On enter Pdf Link open new tab" href="@Url.Action("DownloadFile", "NoticesToMariners", new { fileName = @showFilesResponseModel[i].Filename, batchId = @showFilesResponseModel[i].BatchId, mimeType = @showFilesResponseModel[i].MimeType, frequency ="leisure" })">@showFilesResponseModel[i].FileDescription</a></td>
+                            <td id="filename_@i"><a target="_blank" aria-label="Download file for @showFilesResponseModel[i].FileDescription. opens in new tab" href="@Url.Action("DownloadFile", "NoticesToMariners", new { fileName = @showFilesResponseModel[i].Filename, batchId = @showFilesResponseModel[i].BatchId, mimeType = @showFilesResponseModel[i].MimeType, frequency ="leisure" })">@showFilesResponseModel[i].FileDescription</a></td>
                             <td id="filesize_@i">@showFilesResponseModel[i].FileSizeinKB (@showFilesResponseModel[i].FileExtension)</td>
                             <td class="td-3 justify-content-center" id="download_@i">
-                                <a target="_blank" aria-label="Download File for @showFilesResponseModel[i].FileDescription. Link open new tab" href="@Url.Action("DownloadFile", "NoticesToMariners", new { fileName = @showFilesResponseModel[i].Filename, batchId = @showFilesResponseModel[i].BatchId, mimeType = @showFilesResponseModel[i].MimeType, frequency ="leisure" })">Download</a>
+                                <a target="_blank" aria-label="Download file for @showFilesResponseModel[i].FileDescription. opens in new tab" href="@Url.Action("DownloadFile", "NoticesToMariners", new { fileName = @showFilesResponseModel[i].Filename, batchId = @showFilesResponseModel[i].BatchId, mimeType = @showFilesResponseModel[i].MimeType, frequency ="leisure" })">Download</a>
```

Comment:
We are following same aria label pattern (opens in new tab) across the application if link will open in new tab. so that screen reader will read like (Download file for xyz. opens in new tab link)

Reply:
That's fair enough - when I spoke it out loud earlier it felt like it could be a little more formal, but I agree that it is much better to stick with the convention.

Plus, that convention is more concise (perhaps I should be as well 🙂)